.rubocop.yml

```diff
@@ -19,7 +19,14 @@ Layout/EmptyLinesAroundArguments:
 
 Layout/LineLength:
   Enabled: true
+  
+Layout/TrailingWhitespace:  
+  Exclude:
+    - 'app/models/short_submission.rb' # Empty line isn't whitespace
 
+Lint/ConstantDefinitionInBlock:
```

Comment:
Can you disable only the appropriate line in the code? See https://docs.rubocop.org/rubocop/configuration.html#disabling-cops-within-source-code